pkg/xpu/xpu.go

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 Nordix Foundation.
+// Copyright (c) 2024 Ericsson AB.
+
```

Comment:
this code should just import and use https://github.com/opiproject/godpu library

Reply:
done

Reply:
library imported in line 15 below